docker/docker-windows.go

```diff
@@ -0,0 +1,64 @@
+// +build windows
+
+package docker
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+var (
+	errDockerMachineUnknown = fmt.Errorf("unknown DOCKER_MACHINE_NAME")
+)
+
+func (r *Runner) ProjectDir() string{
+	return "/c/Users/dev/GoPath/src/github.com/evo-cloud/hmake"
```

Comment:
this is in coming refactor, the projectDir is string like `C:\\C\\Users\\***`, should transfer it to `/c/Users/xxx`